specification/cognitiveservices/data-plane/Language/stable/2022-05-01/analyzeconversations.json

```diff
@@ -14,11 +14,11 @@
     "https"
   ],
   "x-ms-parameterized-host": {
-    "hostTemplate": "{endpoint}",
+    "hostTemplate": "{Endpoint}",
```

Comment:
This actually did change the casing. The validation must be buggy. I'd change it back to lowercase. Sorry about that.

Reply:
Wait, nevermind. This was uppercase in the original spec (I was forgetting this was the base of your existing feature branch). So, yeah, this is correct. Ignore the erroneous validation error.

https://github.com/Azure/azure-rest-api-specs/blob/6885351bec8d6a2cea85c5aa793e53616e5f517b/specification/cognitiveservices/data-plane/Language/stable/2022-05-01/analyzeconversations.json#L22